src/robot/parsing/model.py

```diff
@@ -595,20 +595,33 @@ def __iter__(self):
 
 
 class ForLoop(_WithSteps):
+    """
+    Properties:
+
+    - flavor: The value of the 'IN' item, with spaces removed.
+              Typically 'IN', 'INRANGE', 'INZIP', 'INEUMERATE', etc.(?)
+    - range:  True if flavor is 'INRANGE'
+    - vars:   Variables set per-iteration by this loop
+    - items:  Items to loop over (or otherwise parameters, like with INRANGE)
+    - comment: a comment, or None.
+    - steps:  A list of steps in the loop.
+    """
```

Comment:
+1 for docstrings. We've generally used them too little especially with internal code.

I'd prefer docstrings to be in [Sphinx](http://sphinx-doc.org/) format, though, because that's what we use to document public APIs. I couldn't quickly find a better doc that would explain how to document params than http://sphinx-doc.org/domains.html#info-field-lists, but you can also see code e.g. in `robot.model` package for an example.